tests/unit/scoring/conftest.py

```diff
@@ -62,23 +62,14 @@ def mfs(spectra) -> tuple[MolecularFamily, MolecularFamily, MolecularFamily]:
     return mf1, mf2, mf3
 
 
-@fixture(scope="module")
+@fixture(scope="session")
 def datalinks(gcfs, spectra, mfs, strains) -> DataLinks:
     """DataLinks object. See `test_data_links.py` for its values."""
     return DataLinks(gcfs, spectra, mfs, strains)
 
 
-@fixture(scope="module")
-def linkfinder(datalinks) -> LinkFinder:
-    """LinkFinder object. See `test_link_finder.py` for its values."""
-    linkfinder = LinkFinder()
-    linkfinder.calc_score(datalinks, link_type="spec-gcf")
-    linkfinder.calc_score(datalinks, link_type="mf-gcf")
-    return linkfinder
-
-
-@fixture(scope="module")
-def npl(gcfs, spectra, mfs, strains, tmp_path_factory) -> NPLinker:
+@fixture(scope="function")
```

Comment:
The unit tests will change the state of `npl` and `mc`, so to keep each unit tests independent, it's better to use scope `function` instead of `module`.